examples/gym/agent.py

```diff
@@ -169,7 +170,8 @@ def react(self) -> None:
             envelope = self.inbox.get_nowait()  # type: Optional[Envelope]
             if envelope is not None:
                 if envelope.protocol_id == 'gym':
-                    gym_msg = GymSerializer().decode(envelope.message)
+                    gym_msg = GymSerializer().decode(envelope.message)  # type: Message
+                    gym_msg = cast(GymMessage, gym_msg)
```

Comment:
Note, @marcofavorito I expect this to be an issue across all skills so we will likely want to move the type casting down to the message object before the return statement.